src/phoenix/trace/span_evaluations.py

```diff
@@ -152,6 +157,24 @@ def __init_subclass__(
             tuple(sorted(prod)) for prod in product(*cls.index_names.keys())
         )
 
+    def to_parquet(self, path: Union[str, Path]) -> None:
+        table = Table.from_pandas(self.dataframe)
+        table = table.replace_schema_metadata(
+            {
+                **table.schema.metadata,
```

Comment:
nit. just to be defensive
```suggestion
                **(table.schema.metadata or {}),
```